repo/packages/H/hdfs/0/marathon.json

```diff
@@ -4,12 +4,12 @@
   "mem": 512,
   "disk": 100,
   "instances": 1,
-  "cmd": "{{#hdfs.custom-config}}echo {{hdfs.custom-config}} | base64 --decode > mesos-site.xml && {{/hdfs.custom-config}}cd hdfs-mesos* && if [ -e '../mesos-site.xml' ]; then cp -f ../mesos-site.xml ./etc/hadoop/; fi && ./bin/hdfs-mesos",
```

Comment:
We should avoid hard coding LD_LIBRARY_PATH and use configuration values similar to what is done in kafka, i.e. {{hdfs.java-home}}